plugins/modules/swim_workflow_manager.py

```diff
@@ -565,23 +619,33 @@ def site_exists(self, site_name):
         site_exists = False
         site_id = None
         response = None
+
         try:
-            response = self.dnac._exec(
-                family="sites",
-                function='get_site',
-                op_modifies=True,
-                params={"name": site_name},
-            )
+            if self.dnac_version <= self.version_2_3_5_3:
+                response = self.dnac._exec(
+                    family="sites",
+                    function='get_site',
+                    op_modifies=True,
+                    params={"name": site_name},
+                )
+                if response:
+                    self.log("Received API response from 'get_site': {0}".format(str(response)), "DEBUG")
+                    site = response.get("response")
+                    site_id = site[0].get("id")
+                    site_exists = True
+            else:
+                response = self.get_sites(site_name)
```

Comment:
Your get_sites also returns None, but we are not doing any None check here..

Reply:
updated the code

Reply:
get_site and get_sites get site_name as a arg, we can use get_site API in dnac.py and use the version logic inside that API. If we do so, we don't need to check this version whenever we call get_site.